source/docs/software/basic-programming/git-getting-started.rst

```diff
@@ -22,16 +24,16 @@ You have to download and install Git from the following links:
 Git Vocabulary
 --------------
 
-Git is composed of several primary elements:
+Git revolves around several core commands:
 
 - **Repository:** the data structure of your code, including a ``.git`` folder in the root directory
-- **Commits:** a particular saved state of the repository, this includes all files and additions
-- **Branches:** a means of separating various commits, having a unique history. This is primarily used for separating development and stable branches.
+- **Commit:** a particular saved state of the repository, this includes all files and additions
+- **Branch:** a means of separating various commits, having a unique history. This is primarily used for separating development and stable branches.
 - **Push:** update the remote repository with your local changes
 - **Pull:** update your local repository with the remote changes
-- **Cloning:** retrieving a local copy of a repository to modify
-- **Forking:** duplicating a pre-existing repository to modify, and to compare against the original
-- **Merging:** combining various changes from different branches/commits/forks into a single history
+- **Clone:** retrieving a local copy of a repository to modify
```

Comment:
Change these to define the noun, not the verb

Reply:
Are you looking at the correct history? They've been changed to their noun form.

Reply:
The word "clone" has, but the definitions define different parts of speech. For commit, you define what a commit is, rather that what committing is, but for clone, you define what cloning is, rather than what a clone is